R/LearnerFTTransformer.R

```diff
@@ -0,0 +1,168 @@
+
+#' @title FT-Transformer
+#' @description
+#' Feature-Tokenizer Transformer for tabular data that can either work on [`lazy_tesor`] inputs
+#' or on standard tabular features.
+#' @references
+#' `r format_bib("gorishniy2021revisiting")`
+#' @export
+LearnerTorchFTTransformer = R6Class("LearnerTorchFTTransformer",
+  inherit = LearnerTorch,
+  public = list(
+    #' @description
+    #' Creates a new instance of this [R6][R6::R6Class] class.
+    initialize = function(task_type, optimizer = NULL, loss = NULL, callbacks = list()) {
+      private$.block = PipeOpTorchFTTransformerBlock$new()
```

Comment:
check that the default arguments are as defined in the paper

Reply:
https://github.com/yandex-research/rtdl-revisiting-models/blob/e3ed46cac38568785289d8fa16b8cfa585bde27e/package/rtdl_revisiting_models.py#L752-L778